pagefaultgames/pokerogue
# Admin panel beta

## What are the changes the user will see?
Admins now have an admin panel to play with

## Why am I making these changes?
[See here](https://discord.com/channels/1125469663833370665/1176874654015684739/1277430244277616651)

## What are the changes from a developer perspective?
Previous admin panel work was done; this PR just updates it to have a proper UI to link PR usernames to discord IDs

### Screenshots/Videos
See [here](https://discord.com/channels/1125469663833370665/1176874654015684739/1277891070562271272)

## How to test the changes?
Unsure for now - can't test api post changes locally
Some things may need fixing - I haven't tested the stuff actually works properly, and there may be some ui oddities because of that
Also haven't run tests to make sure they all still work

## Checklist
